Doc/library/calendar.rst

```diff
@@ -393,13 +393,22 @@ The :mod:`calendar` module exports the following data attributes:
 
 .. data:: day_name
 
-   An array that represents the days of the week in the current locale.
+   A sequence that represents the days of the week in the current locale,
+   where ``Monday`` is ``0`` and ``Sunday`` is ``6``.
```

Comment:
I think following the convention of month_name and day_abbr would be better, so this would become: "where Monday is day number 0", and omitting mention of Sunday.